src/runtime/cli/config-generated.go.in

```diff
@@ -36,6 +36,7 @@ var version = "@VERSION@"
 // project-specific command names
 var envCmd = fmt.Sprintf("%s-env", projectPrefix)
 var checkCmd = fmt.Sprintf("%s-check", projectPrefix)
+var execCmd = "exec"
```

Comment:
I don't think you need this change? This file is for code generated by the build, but this variable isn't generated by the build :smile: